src/Tribe/Repositories/Attendee/Commerce.php

```diff
@@ -105,4 +105,103 @@ public function checked_in_keys() {
 		return $this->limit_list( $this->key_name, parent::checked_in_keys() );
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	public function update_additional_data( $attendee, $ticket, $attendee_data ) {
+
+		$attendee_data = $this->format_attendee_data( $attendee, $ticket, $attendee_data );
+
+		try {
+			$query = $this->by_primary_key( 'id', $attendee->ID )
+			              ->set_args( $attendee_data );
```

Comment:
* :warning: Found precision alignment of 2 spaces.
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 00fc956183107f92c0431ec0a5f36c04</sub>